R/wl_convert_units.R

```diff
@@ -0,0 +1,207 @@
+
+#' Convert between Different Wavelength Units
+#'
+#' The following units can be converted into each other:
+#' *nm*, \emph{\eqn{cm^{-1}}{inverse cm}}, *eV*, *THz* and *Raman shift*
+#'
+#' @param x data for conversion
+#' @param from source unit
+#' @param to destination unit
+#' @param ref_wl laser wavelength (required for work with Raman shift)
+#' @author R. Kiselev
+#' @export
+#'
+#' @concept wavelengths
+#'
+#' @examples
+#' wl_convert_units(3200, "Raman shift", "nm", ref_wl = 785.04)
+#' wl_convert_units(785, "nm", "invcm")
+wl_convert_units <- function(x, from, to, ref_wl = NULL) {
```

Comment:
@sangttruong, @bryanhanson  Do I understand correctly, that this file `wl_convert_units.R` was moved/copied from `hyperSpec` to `hySpc.read.spe`? What is the purpose to do so? And why copying/moving was preferred over exporting?

Reply:
I think this function must be left in `hyperSpec` and imported here using `#' @importFrom hyperSpec  wl_convert_units`.  I'll add a separate comment where is a good place to add this `roxygen2` comment.

We may discuss this if needed.

Reply:
Addressed in #3

Reply:
@bryanhanson